README.md

```diff
@@ -11,7 +11,6 @@ Get a minimal rails 5.1+ app ready to be deployed on Heroku with Bootstrap, Simp
 ```bash
 rails new \
   --database postgresql \
-  --webpack \
```

Comment:
We want to keep this as the templates can be used by students who don't have rails 6. In the slides it's different because the slides are maintained for a "during bootcamp" context, whereas the templates are also maintained for an "after bootcamp" context 👌

Reply:
Got it, sorry I didn't think of it